public/locales/uk.json

```diff
@@ -25,6 +25,7 @@
   "yourHistory": "Ваша історія",
   "scrobbleManually": "Скробблити вручну",
   "scrobbleFromOtherUser": "Скробблити за іншим користувачем",
+  "scrobbleFromSetlist": "Скробблити зі списку",
```

Comment:
I would suggest "Скробблити з setlist", to emphasize that user's going to scrobble from a service called "setlist". Otherwise in the current form it basically says "scrobble from a list", which makes too little sense.

Reply:
Thanks for the tip! If it wasn't obvious these were all just google translate attempts so I appreciate real human help! :)

Reply:
FYI, the project translations are handled through POEditor, so don't worry too much about it for the moment. Everyone will be able to contribute and improve translations once we add them there!